File: LocalAPI/js/env/clear-env-cache.js
const axios = require('axios');

const config = {
  method: 'post',
  url: 'http://localhost:60000/api/v2/env/clearCache',
  headers: { },
  data: {
    id: "181",
    cacheList: ["indexedDB", "localStorage", "cookies"],
    clearEnvFolder: 0
  }
};

axios(config)
.then((response) => {
  console.log(JSON.stringify(response.data));
})
.catch((error) => {
  console.log(error);
}); 

File: LocalAPI/python/env/clear_env_cache.py
import requests

data = {
    'id': '181',
    'cacheList': ['indexedDB', 'localStorage', 'cookies'],
    'clearEnvFolder': 0
}

response = requests.post('http://localhost:60000/api/v2/env/clearCache', json=data)
print(response.text) 